src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Application {
+    public String getRandomBall() {
+
+        Random random = new Random();
```

Comment:
lets make Random a class level variable 
lets create also a ColoSupplier, get a random color and set it to the ball that you should return from this method